static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -350,6 +351,25 @@ function OnboardingContent({
     );
   }
 
+  // if the org cannot ingest mobile replay events, don't show the onboarding
+  // TODO: remove once we GA mobile replay
+  if (organization.features.includes('session-replay-video-disabled')) {
```

Comment:
`false` means the org can ingest mobile replays.

Reply:
yup that's the intended result! if the flag is true we want to show these banners

Reply:
Nevermind. Its done correctly.